tests/conftest.py

```diff
@@ -10,7 +10,7 @@
 # redis 6 release candidates report a version number of 5.9.x. Use this
 # constant for skip_if decorators as a placeholder until 6.0.0 is officially
 # released
-REDIS_6_VERSION = '5.9.0'
+REDIS_6_VERSION = '6.2.0'
```

Comment:
This is not correct. `REDIS_6_VERSION` actually means "Since which version Redis6 feature introduced". Please check the following usage.
`
@skip_if_server_version_lt(REDIS_6_VERSION)
`

So you cannot change it to `6.2.0`

Reply:
You are right, this is a poor work around. The issue is that I do not understand why the tests are failing with Redis 6.2. All I want here is to add the _getdel_ verb that is now available in Redis. I would appreciate a lot some help to resolve the Unit Test issues of that PR. If not I will cancel it and wait for somebody else to add _getdel_ in another PR.